gradle/libs.versions.toml

```diff
@@ -1,19 +1,19 @@
 [versions]
-commons-io = "2.11.0"
-download-task = "4.1.1"
+commons_io = "2.11.0"
+download_task = "4.1.1"
 guava = "31.1-jre"
-jackson-xml = "2.13.4"
-launchermeta-parser = "1.0.0"
-enigma = "1.2.2"
-enigma-plugin = "1.2.1"
-tiny-remapper = "0.7.2"
+jackson_xml = "2.14.1"
+launchermeta_parser = "1.0.0"
+enigma = "1.3.1"
+enigma_plugin = "1.2.1"
+tiny_remapper = "0.7.2"
 stitch = "0.6.1"
 unpick = "3.0.2"
 mapping-io = "0.3.0"
-javadoc-draftsman = "1.2.1"
+javadoc_draftsman = "1.2.1"
 
-fabric-loader = "0.14.10"
-jetbrains-annotations = "23.0.0"
+fabric_loader = "0.14.11"
```

Comment:
Hmm, now I'm wondering why we still have FLoader instead of QLoader (yes, it's because the JAR merger uses its env type stuff, but why FLoader ;-;)
I'd say leave it alone for now, we'll get it on another day :P

Reply:
on that note, stitch is also marked under `net.fabricmc:stitch`, and stitch isn't on our maven, which leads me to believe we aren't even using our fork :p